src/components/CollapsibleCard/CollapsibleCard.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+// Components
+import Card from '../Card/Card';
+
+// Style
+import style from './style.scss';
+
+const styleColors = { RED: 'red' };
+
+class CollapsibleCard extends Component {
+  state = {
+    displayCard: false,
+  }
+
+  onClick = () => {
+    this.setState({ displayCard: !this.state.displayCard });
+  }
+
+  render() {
+    const {
+      headerText,
+      expandableText,
+      children,
+      styleColor,
+      disabled,
+    } = this.props;
+
+    const {
+      displayCard,
+    } = this.state;
+
+    const collapsibleCardStyle = cx(
+      style.collapsibleCard,
+      {
+        [style.styleColorRed]: styleColor === styleColors.RED,
+      }
+    );
+
+    return (
+      <div className={collapsibleCardStyle}>
+        <div className={style.cardHeader}>
+          {headerText}
+          {!disabled && (
+            <div className={style.textLink} onClick={this.onClick}>
+              <b>{expandableText}</b>
```

Comment:
Should the text always be bold?

Reply:
That is a good point; it may not need to always be bold. I'll remove expandable text always being bold. If it needs to be bold, it can be passed into the component that way.